testsuite/features/secondary/min_docker_api.feature

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2017-2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+#
+# Note: image building via API is not tested here
+#       it is tested in buildhost_docker_build_image.feature and
+#                       buildhost_docker_auth_registry.feature
+
+@scope_building_container_images
+Feature: API "image" namespace for containers and sub-namespaces
+
+  Scenario: Test "image.store" namespace
+    When I am logged in API as user "admin" and password "admin"
```

Comment:
I would move login to another scenario above that one.
And logout to another scenario below the one where it is, as... we are mixing what the scenario name says, having login in one scenario and logout in another.

Reply:
Can we just agree on create a card or a checkbox in the current card to don't forget about it?

Reply:
i'll create a card